### DISCUSSION TITLE
Github Desktop For Linux?

### DISCUSSION THREAD

**Discussion by angelwood-xyz** [community/community#22660] [New to GitHub]
<p>I am just Wondering if there is a linux version of the "Github Desktop" application</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>We don’t have an officially supported GitHub Desktop for any linux distros at present. That said, there is a community maintained project you might find exciting:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/shiftkey/desktop" target="_blank" rel="noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/11161530/181072527-caa6ce55-52f9-4db5-8677-f4f54844bfca.jpeg" class="thumbnail onebox-avatar" width="343" height="343">

<h3><a href="https://github.com/shiftkey/desktop" target="_blank" rel="noopener">shiftkey/desktop</a></h3>

<p>Fork of GitHub Desktop to support various Linux distributions - shiftkey/desktop</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by angelwood-xyz** [comment]
<p>Work Perfectly, Thanks :) </p>
